File: tests/fixtures/lang/sample.rs
/// Adds two numbers.
pub fn add(x: i32, y: i32) -> i32 {
    // sum them
    x + y
}

pub struct Widget {
    count: i32,
}

impl Widget {
    /// Renders the widget.
    pub fn render(&self, label: &str) -> String {
        label.to_string()
    }

    fn hidden(&self) {}
}

pub const MAX_WIDGETS: i32 = 10;

File: tests/fixtures/lang/sample.cs
using System;

namespace Frob.Sample
{
    /// <summary>Adds two numbers.</summary>
    public class Widget
    {
        public int Count { get; set; }

        private int hidden;

        public const int MaxWidgets = 10;

        /// <summary>Renders the widget.</summary>
        public string Render(string label)
        {
            // sum them
            return Add(label);
        }

        private string Add(string label)
        {
            return label;
        }
    }

    internal interface IThing
    {
        void Do();
    }

    public enum Color
    {
        Red,
        Green,
        Blue,
    }
}
